ttml.go

```diff
@@ -347,8 +352,11 @@ func ReadFromTTML(i io.Reader) (o *Subtitles, err error) {
 	// Loop through subtitles
 	for _, ts := range ttml.Subtitles {
 		// Init item
+		ts.Begin.tickrate = ttml.Tickrate
```

Comment:
Can you move this line just below `ts.Begin.framerate = ttml.Framerate` ? (alphabetical order)